Documentation/Scripts/generateMdFiles.py

```diff
@@ -146,12 +147,14 @@ def getRestBodyParam():
 def getRestDescription():
     #print >>sys.stderr, "RESTDESCRIPTION"
     if thisVerb['description']:
-        #print >> sys.stderr, thisVerb['description']
-        return RX3[0].sub(RX3[1], thisVerb['description'])
+        description = thisVerb['description']
+        #print >> sys.stderr, description
```

Comment:
use `print(*args, file=sys.stderr)`, `sys.stderr.write()` or better the logging module this does not work in python3 and it has to be ported to oskar